Testscripts/Linux/validate-da.sh

```diff
@@ -58,18 +64,25 @@ function verify_directory_exists() {
     done
 
     if $failed_assertions; then
-        LogErr "Some/All directories exists"
-        echo "$failed_assertions"
+        LogErr "Some directories exists"
+        $failed_assertions
     fi
 }
```

Comment:
As written, when `$failed_assertions` the exit status of the function is not obvious. Always evaluate `$failed_assertions` as the last statement in the function to ensure the exit status of the function is deterministic.

Reply:
Done